src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--31">
```

Comment:
The class 'calendar--31' suggests that the calendar will always have 31 days. Instead, consider making the number of days in the calendar dynamic based on the month.